extensions/eclipse-che-theia-plugin-remote/src/browser/remote-plugin-command.ts

```diff
@@ -0,0 +1,32 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { CommandRegistry, CommandContribution, Command } from '@theia/core/lib/common';
+import { RemotePluginStarterService } from '../common/remote-plugin-protocol';
+
+const COMMAND: Command = {
+    id: '_remote_plugin_start_'
```

Comment:
Could you please make the command ID consistent with other Theia command and name it something like `remote.plugin.start` ( or `remote.plugins.start` if it handles a set of plugins )